docs/toc.yml

```diff
@@ -420,6 +418,9 @@ items:
         - name: Concepts (preview)
           href: excel/excel-data-types-concepts.md
           displayName: Excel
+        - name: Entity value cards (preview)
```

Comment:
I didn't think we included "(preview)" in the TOC. Did that change?

Reply:
I have been including it for the data types articles because the entire feature is still in preview. Open to removing it if that's inconsistent with our process though

Reply:
Discussed offline, will remove (preview) from TOC in a subsequent commit.